header.h

```diff
@@ -0,0 +1,141 @@
+#include <iostream>
+#include <string>
+
+using namespace std;
+
+class BankAccount
+{
+private:
+
+
+    string Owner_name;
```

Comment:
Don't use capital case in class data members 
Use something like this `string owner_name`

Reply:
Use this to all your varriable names